content/imgpkg/docs/latest/basic-workflow.md

```diff
@@ -11,19 +11,19 @@ You want to create an immutable artifact containing Kubernetes configuration and
 To complete this workflow you will need access to an OCI registry like Docker Hub, and optionally, 
 a Kubernetes cluster. (If you would like to use a local registry and Kubernetes cluster, try using [Kind](https://kind.sigs.k8s.io/docs/user/local-registry/))
 
-If you would like to deploy the results of this scenario to your Kubernetes cluster, you will additionally need [`kbld`](https://carvel.dev/kbld/) and kubectl.
+If you would like to deploy the results of this scenario to your Kubernetes cluster, you will additionally need [`kbld`](/kbld) and kubectl.
 
 ## Step 1: Creating the bundle
 
 1. Prepare bundle contents
 
-    The [examples/basic-step-1/](../examples/basic-step-1) directory has a `config.yml` file, which contains a very simple Kubernetes application. Your application may have as many configuration files as necessary in various formats such as plain YAML, ytt templates, Helm templates, etc.
+    The [examples/basic-step-1/](https://github.com/vmware-tanzu/carvel-imgpkg/tree/develop/examples/basic-step-1) directory has a `config.yml` file, which contains a very simple Kubernetes application. Your application may have as many configuration files as necessary in various formats such as plain YAML, ytt templates, Helm templates, etc.
```

Comment:
imgpkg docs are under https://carvel.dev/imgpkg/docs/latest/. these shouldnt be pointing at github.

Reply:
these are links to examples that I could only find in the imgpkg github, I did not see any markdown files under impkg/docs/latest in the carvel.dev repo that corresponded to these examples.

Reply:
ah you are absolutely right. my bad.